docker/api/client.py

```diff
@@ -7,6 +7,19 @@
 import six
 import websocket
 
+from .. import auth
```

Comment:
where do these changed order of these imports come from? Was that intentional? If not needed, better to keep it separate and focus the PR on just the "auto" change

Reply:
My IDE is sorting imports on save. Could you ignore the reordering please, it is not bad having them sorted anyway.

Reply:
I see it's been merged, but it's really better to keep these changes separate (at _least_ a separate commit). In this commit, more than half of the line changes is now actually unrelated to the change itself.

I also see that the second commit was to fix the tests to adjust for the changes of the first commit. Order of the commits should have been reversed to not break `git bisect`